Difi.SikkerDigitalPost.Klient.Tester/SmokeTestsHelper.cs

```diff
@@ -22,9 +25,22 @@ internal class SmokeTestsHelper
         private Transportkvittering _transportkvittering;
         private Forretningskvittering _forretningskvittering;
 
-        public SmokeTestsHelper(Miljø miljø)
+        public SmokeTestsHelper(Miljø miljø, bool loggHttp = false, bool brukProxy = false)
         {
-            _klient = new SikkerDigitalPostKlient(new Databehandler(DomainUtility.Organisasjonsnummer(), DomainUtility.GetAvsenderCertificate()), new Klientkonfigurasjon(miljø));
+            var config = new Klientkonfigurasjon(miljø);
+
+            config.LoggForespørselOgRespons = loggHttp;
+            
+            if (brukProxy)
+            {
+                config.ProxyHost = "127.0.0.1";
+                config.ProxyPort = 8888;
+                config.ProxyScheme = "http";
+                config.TimeoutIMillisekunder = 2000;
+            }
```

Comment:
jeg vet ikke helt hvor denne kalles med brukProxy, hadde det vært bedre om man hadde sendt med et proxyconfig objekt elns i stedet for a hardkode proxyconfigurasjonen for denne ende-til-ende-testen?